amundsen_application/api/preview/v0.py

```diff
@@ -29,17 +27,18 @@
 
 @preview_blueprint.route('/', methods=['POST'])
 def get_table_preview() -> Response:
-    # TODO: Want to further separate this file into more blueprints, perhaps a Blueprint class is the way to go
     global PREVIEW_CLIENT_INSTANCE
+    global PREVIEW_CLIENT_CLASS
     try:
-        if PREVIEW_CLIENT_INSTANCE is None and PREVIEW_CLIENT_CLASS is not None:
-            PREVIEW_CLIENT_INSTANCE = PREVIEW_CLIENT_CLASS()
-
         if PREVIEW_CLIENT_INSTANCE is None:
-            payload = jsonify({'previewData': {}, 'msg': 'A client for the preview feature must be configured'})
-            return make_response(payload, HTTPStatus.NOT_IMPLEMENTED)
+            if (app.config['PREVIEW_CLIENT_ENABLED']
+                    and app.config['PREVIEW_CLIENT'] is not None):
+                PREVIEW_CLIENT_CLASS = import_string(app.config['PREVIEW_CLIENT'])
```

Comment:
may want to add an exception handle in case it doesn't work.

Reply:
`import_string` raises an error, not an exception, if the module cannot be located. So no can do :(